src/backend/mpi/swdsm.cpp

```diff
@@ -411,10 +412,10 @@ void handler(int sig, siginfo_t *si, void *unused){
 	state  = cacheControl[startIndex].state;
 	tag = cacheControl[startIndex].tag;
 
-	if(state == INVALID || (tag != lineAddr && tag != GLOBAL_NULL)){
-		load_cache_entry(alignedDistrAddr, (startIndex%cachesize));
+	if(state == INVALID || (tag != aligned_access_offset && tag != GLOBAL_NULL)) {
+		load_cache_entry(aligned_access_offset, (startIndex%cachesize));
 #ifdef DUAL_LOAD
-		prefetch_cache_entry((alignedDistrAddr+CACHELINE*pagesize), ((startIndex+CACHELINE)%cachesize));
+		prefetch_cache_entry((aligned_access_offset+CACHELINE*pagesize), ((startIndex+CACHELINE)%cachesize));
```

Comment:
I notice that `alignedDistrAddr` and `lineAddr` seem to have been merged in the new code, I assume it's on purpose?

Reply:
see line 324 (before the patch): `alignedDistrAddr = lineAddr;`
This is the alias that @pekemark complained about in #26

Reply:
Ah right, I missed that.